### ISSUE TITLE
Does status page indicate if server connection is lost?

### ISSUE THREAD

**Issue by dgleba** [louislam/uptime-kuma#1360]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

On the dashboard, if the connection to the uptime-Kuma server is lost, a message appears stating such.

`Cannot connect to the socket server. [Error: websocket error] Reconnecting...`

I don't see an indication on the `status page` if the uptime-Kuma server is not healthy.
I think the page would simply continue to say everything is working OK and it would not update.

Am I incorrect?

### 🐻 Uptime-Kuma Version

1.12.1

### 💻 Operating System and Arch

ubuntu 20.04 x86

### 🌐 Browser

Chrome 98

### 🐋 Docker Version

Docker 20.10.12

**Comment by louislam** [comment]
Yes, there is no such error, because status page is using REST api instead of WebSocket.

**Comment by vanderlpp** [comment]
I'm having the same problem, but only on the dashboard.
on the status page is ok.
I'm running the system in a VM on proxmox, on a local server, at the office, and releasing the port for remote access in the subdomain [http://monitor.esc.adv.br](http://monitor.esc.adv.br/
)
whats wrong?




🐻 Uptime-Kuma Version
1.11.4 

💻 Operating System and Arch
ubuntu ubuntu-20.04-standard_20.04-1_amd64

🌐 Browser
Chrome 99.0.4844.51

🐋 Docker Version
NO Docker 
![Captura de tela 2022-03-11 122615](https://user-images.githubusercontent.com/28407156/157897144-01c9275a-c23e-4a75-9e84-576735df01d5.jpg)


🟩 NodeJS
![Captura de tela 2022-03-11 122654](https://user-images.githubusercontent.com/28407156/157897149-cc22cd5a-51ca-4ed7-a1a0-8c81fdc9d3e0.jpg)
 Version
No respons
![Captura de tela 2022-03-11 122948](https://user-images.githubusercontent.com/28407156/157897619-d7840827-36e3-42de-9494-5f7a81452bd4.jpg)
e

**Comment by louislam** [comment]
> I'm having the same problem, but only on the dashboard. on the status page is ok. I'm running the system in a VM container in proxmox, on a local server, at the office, and releasing the port for remote access in the subdomain [http://monitor.esc.adv.br](http://monitor.esc.adv.br/) Using reverse proxy to respond exactly to only the configured subdomain. whats wrong?
> 
> 🐻 Uptime-Kuma Version 1.11.4
> 
> 💻 Operating System and Arch ubuntu ubuntu-20.04-standard_20.04-1_amd64
> 
> 🌐 Browser Chrome 99.0.4844.51 ![Captura de tela 2022-03-11 123523](https://user-images.githubusercontent.com/28407156/157898546-5e188b52-f594-498d-afc1-1db2baa54a1f.jpg)
> 
> 🐋 Docker Version NO Docker ![Captura de tela 2022-03-11 122615](https://user-images.githubusercontent.com/28407156/157897144-01c9275a-c23e-4a75-9e84-576735df01d5.jpg)
> 
> 🟩 NodeJS ![Captura de tela 2022-03-11 122654](https://user-images.githubusercontent.com/28407156/157897149-cc22cd5a-51ca-4ed7-a1a0-8c81fdc9d3e0.jpg) Version No respons ![Captura de tela 2022-03-11 122948](https://user-images.githubusercontent.com/28407156/157897619-d7840827-36e3-42de-9494-5f7a81452bd4.jpg) e

You unchecked "Enable WebSocket".

**Comment by vanderlpp** [comment]
> 
> Você desmarcou "Ativar WebSocket".

Where is this option?

**Comment by Henk57** [comment]
It has cost me a lot of time to find out what was wrong. I use a Synology NAS with docker installled, DSM 7.
In my case I had to add Custom Headers to the reverse proxy rules. (I've multiple services running on 1 IPv4 address)
It was then default suggested by DSM in the 'reverse proxy rules - custom headers' tab for that specific item:

Upgrade  -  $http_upgrade
Connection - $conection_upgrade

And after that the "cannot connect to the socket server." error was gone, and I got the normal login screen. An that worked.
